pandas/tests/extension/base/setitem.py

```diff
@@ -1,6 +1,15 @@
 import numpy as np
 import pytest
 
+try:
+    import uncertainties.unumpy as unp
```

Comment:
this is not appropriate for the pandas test suite as this is not in any way tested in ci

Reply:
Thanks for the review!  Do you have any suggestions as to what test cases I should look at to see how other extension packages deal with base tests that don't work as written?  I've seen some examples where the hooks are there to create the correct sort of data (using fixtures) or hook the right functions (so that base test code calls extension functions).  But am new to how I should deal with this case.  If you guide, I can implement, test, and amend.